content/docs/reference-react-component.md

```diff
@@ -635,7 +635,7 @@ The `displayName` string is used in debugging messages. Usually, you don't need
 
 `this.props` contains the props that were defined by the caller of this component. See [Components and Props](/docs/components-and-props.html) for an introduction to props.
 
-In particular, `this.props.children` is a special prop, typically defined by the child tags in the JSX expression rather than in the tag itself.
```

Comment:
It's still important to include the "rather than the tag itself" because `<div children={<span>Hello, Dave!</span>} />` is also possible (typically when spreading props). We shouldn't omit that possibility.